content/release-10-16-0/devices-10-16-0-bundle/10-16-0-0.md

```diff
@@ -95,6 +119,62 @@ Device Management</td>
 DM-1228</td>
 </tr>
 
+<tr>
+<td>
+Device Management</td>
+<td>When deleting devices in the device list view, the options to delete child devices and device owners are again available in the confirmation window.</td>
+<td>
+DM-1319</td>
+</tr>
+
+<tr>
+<td>
+Device Management</td>
+<td>Misleading tooltips in the <b>Groups</b> page have been removed.</td>
+<td>
+MTM-47241</td>
+</tr>
+
+<tr>
+<td>
+Device Management</td>
+<td>In the Device Management application, for example in the <b>Info</b> tab or the <b>Location</b> tab, the latitude position is now consistently displayed as first position value, followed by longitude.</td>
+<td>
+MTM-41052</td>
+</tr>
+
+<tr>
+<td>
+Device Management</td>
+<td>The "Asset properties" widget's data type mapping has been improved to ignore null values for properties created by default when using MQTT templates, which prevented user from saving widget data unless the mentioned properties were fixed manually. Affected properties: c8y_Firmware.url, c8y_Software.url, c8y_Mobile.imei, c8y_Mobile.imsi, c8y_Mobile.cellId, c8y_Mobile.lac, c8y_Mobile.mcc, c8y_Mobile.mnc.</td>
```

Comment:
not sure about the parentheses here

Reply:
This is the name of the asset type, always written in quotes. I will rephrase